### PR TITLE
Fix content view filter api tests according to 6.2

### PR DESCRIPTION
**Pull request by oshtaier**
Refer to https://github.com/SatelliteQE/robottelo/pull/3449 for more details
```
nosetests tests/foreman/api/test_contentviewfilter.py -m test_negative_get
..
----------------------------------------------------------------------
Ran 2 tests in 31.887s

OK
```

### CODE REVIEW

File: tests/foreman/api/test_contentviewfilter.py [hunk 1]
```diff
@@ -44,20 +44,15 @@ def test_negative_get_with_no_args(self):
 
         @Feature: ContentViewFilter
 
-        @Assert: An HTTP 400 or 422 response is received if a GET request is
-        issued with no arguments specified.
-
-        This test targets bugzilla bug #1102120.
+        @Assert: An HTTP 200 response is received if a GET request is issued
+        with no arguments specified.
         """
```
Comment (sthirugn) [inline tests/foreman/api/test_contentviewfilter.py]: can you please add a statement here:
```
# see bz 1102120.
```

File: tests/foreman/api/test_contentviewfilter.py [hunk 2]
```diff
@@ -66,21 +61,16 @@ def test_negative_get_with_bad_args(self):
 
         @Feature: ContentViewFilter
 
-        @Assert: An HTTP 400 or 422 response is received if a GET request is
-        issued with bad arguments specified.
-
-        This test targets bugzilla bug #1102120.
+        @Assert: An HTTP 200 response is received if a GET request is issued
+        with bad arguments specified.
         """
```
Comment (sthirugn) [inline tests/foreman/api/test_contentviewfilter.py]: same as above

**Comment by sthirugn** [comment]
ACK pending non blocker comment

**Comment by oshtaier** [comment]
Actually 1102120 BZ will only confuse a person as behaviour changed twice and not to that we have right now

**Comment by sthirugn** [comment]
ah ok please ignore my comment then :)

**Pull request merged**